usecases/nodes/handler.go

```diff
@@ -52,6 +53,11 @@ func (m *Manager) GetNodeStatus(ctx context.Context,
 	if err := m.authorizer.Authorize(principal, authorization.READ, authorization.Cluster()); err != nil {
 		return nil, err
 	}
+	if verbosity == verbosityPkg.OutputVerbose {
+		if err := m.authorizer.Authorize(principal, authorization.READ, authorization.CollectionsData(className)...); err != nil {
+			return nil, err
+		}
+	}
```

Comment:
My idea was more that we require special permission for verbose=true vs false. But am not sure if that is easily possible 🤔  Reason being that the call is SUPER expensive on large cluster

I am not 100% sure what info is in the nodes API with and without verbose.

Reply:
That's a good idea in addition to what I have there

Having `output=verbose` will return `[]*NodeShardStatus` and `*NodeStats` that include metadata and data about all the classes, hence the authorization call there. Although, now I mention this that should be `Collections(className)` since the names of the classes come back too in addition to the object count, etc.

We could introduce a resource here for cluster and allow `*, verbose, minimal`? So something like:
```
cluster/nodes/verbose
```
and a new field to `Permission` like:
```
"cluster": {
  "nodes": "verbose"
}
```
wdyt?

Reply:
yep, that sounds good!